Python/Watermark_Images/watermark_logo.py

```diff
@@ -0,0 +1,42 @@
+import cv2
+import numpy as np
+
+#importing the main image
+pImg = input("Enter image path: ")
+image = cv2.imread(pImg)
+oH,oW = image.shape[:2]
+image = np.dstack([image, np.ones((oH,oW), dtype="uint8") * 255])
+
+

```

Comment:
I was just wondering, if we can have some modularity in the script? Can you make a function for this and call it to run the script.

Similarly for `watermark_text.py`

Reply:
Done!